projects/packages/stats-admin/src/class-rest-controller.php

```diff
@@ -77,15 +102,8 @@ public function register_rest_routes() {
 				'permission_callback' => array( $this, 'check_user_privileges_callback' ),
 			)
 		);
-		register_rest_route(
-			static::$namespace,
-			sprintf( '/sites/%d', Jetpack_Options::get_option( 'id' ) ),
-			array(
-				'methods'             => WP_REST_Server::READABLE,
-				'callback'            => array( $this, 'site' ),
-				'permission_callback' => array( $this, 'check_user_privileges_callback' ),
-			)
-		);
+
+		// TODO remove call to the endpoint from the frontend.
 		register_rest_route(
```

Comment:
The endpoint is just a place holder for the the frontend to function, meaning it only returns an empty object and doesn't forward anything with blog token to wpcom API.

Reply:
Why not completely remove it to keep our code clean?